src/frontend/screens/JoinRequestModal.tsx

```diff
@@ -107,7 +107,7 @@ export const JoinRequestModal: NavigationStackScreenComponent<{
           ]}
           icon={
             <Circle radius={40}>
-              <MaterialIcon color="#000033" name="person-add" size={30} />
+              <MaterialIcon color="#000033" name="highlight" size={30} />
```

Comment:
is this change supposed to be here? this component refers to [this](https://www.figma.com/file/ZOsM89L0xy0NUOCkuksJgq/OTF-%2B-UXFund?node-id=46972%3A741)

Reply:
Oh, no I was just messing around with something and I forgot to change it back.